cypress/support/commands.js

```diff
@@ -23,3 +23,6 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+Cypress.Commands.add('findById', (id) => {
+  cy.get(`#${id}`);
```

Comment:
The `findById` command should return the result of `cy.get(`#${id}`)` to allow for command chaining. Add a `return` statement before `cy.get` to ensure the command can be used in a chain.